sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -166,6 +164,17 @@ export interface FullEventProcessorOptions
    * passing the data to user code for processing. If not provided, it defaults to 60 seconds.
    */
   maxWaitTimeInSeconds: number;
+
+  /**
+   * The amount of time to wait between each attempt at claiming partitions.
+   */
+  loopIntervalInMs?: number;
+
+  /**
+   * The maximum amount of time since a PartitionOwnership was updated
+   * to use to determine if a partition is no longer claimed.
+   */
+  inactiveTimeLimitInMs?: number;
```

Comment:
I think if something is configurable by the user I can see the argument of setting a default value at the same place the user-provided argument gets passed.

So if `maxWaitTime` can be passed into an `EventHubConsumerClient` and has a default value of 60, I can see having the default set on the client rather defined by the `receiveBatch` call that it eventually gets provided to.

At this point since this isn't a user-configurable option (only our tests set it) I kept the default in this class. Then I also didn't have to add noise to the PR by updating all the EventProcessor tests. When we plan to make this user-configurable I can see the default being set higher up.